src/BaGet.Core/Configuration/HttpsOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace BaGet.Core.Configuration
+{
+    public class HttpsOptions
+    {
+        public int Port { get; set; }
+
+        public string CertificateFileName { get; set; }
+
+        public string CertificatePassword { get; set; }
+    }
+}
```

Comment:
I don't understand the need to add a custom options class. Doesn't kestrel have https configuration options by default? https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.1#listenoptionsusehttps